.github/workflows/release.yml

```diff
@@ -0,0 +1,419 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  contents: write
+
+jobs:
+  # Stage 1: Build Windows binaries (parallel)
+  build-windows-x86:
```

Comment:
I'm not 100% sure due to the artifact handling, but I guess it should be possible to use `strategy.matrix` to define the build-windows, package-windows and installer-windows jobs once, instead of individually for x86, x64 and arm64.

Reply:
Nice catch! I'm still getting the hang of this (and the whole 'strategy matrix' concept). Looking into it now... 👍🏻

Reply:
Done in https://github.com/FirebirdSQL/firebird-odbc-driver/pull/245/commits/ed49e7547d621afe8386741336ded57326c465d0